kuma/users/templates/users/email/welcome/plain.ltxt

```diff
@@ -30,7 +30,7 @@
 {{ _('Want to talk to someone about MDN? There are a few ways you can do that:') }}
 
 {# L10n: This is an email. Whitespace matters! #}
-* {{ _('Mailing list: {list_link}')|f(list_link=add_utm('https://lists.mozilla.org/listinfo/dev-mdc', 'welcome')) }}
+* {{ _('Mailing list:') }} {{ add_utm('https://lists.mozilla.org/listinfo/dev-mdc', 'welcome') }}
```

Comment:
I changed it b/c the link doesn't need to be translated. But I could see it needing to be included for RTL languages.